plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -246,8 +249,12 @@ def spawn_helper(
         prev_checkpoint=checkpoint_src,
     ):
         fn = cloudpickle.loads(fn)
-        return_val = fn(**kwargs)
-
+        try:
+            return_val = fn(**kwargs)
```

Comment:
This is the invocation of the actual task function in the worker processes.

Reply:
lets add a comment here

Reply:
Will do 👍

Reply:
https://github.com/flyteorg/flytekit/pull/1846/commits/5ebce626a6866a071a75fa34f8a1c8dfa491cd5f